presto-main/src/main/java/com/facebook/presto/memory/MemoryPool.java

```diff
@@ -235,6 +270,7 @@ public synchronized void freeRevocable(QueryId queryId, long bytes)
         else {
             queryMemoryRevocableReservations.put(queryId, queryReservation);
         }
+
```

Comment:
Same opinion here. We should keep them matching.

Reply:
Done